### ISSUE TITLE
Authentication required after submission of the Quick start setup wizard form.

### ISSUE THREAD

**Issue by cpauya** [learningequality/kolibri#7030]
### Observed behavior
I followed the [Setting up Kolibri for development](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#setting-up-kolibri-for-development) documentation using the `develop` branch, run Kolibri, and filled the **"Quick start"** setup wizard form.  When submitting, I got a prompt "Authentication Required" for the "api".  

I don't know what account to put in the dialog.  I also saw an HTTP error 401 in the terminal.

Notes: 
 
1. I deleted the `~/.kolibri/` folder for a "fresh" install.
1. This also applies to the `release-v0.14.x` branch as of today.

### Expected behavior
Setup wizard should complete without prompts or errors.

### Errors and logs

Last three (3) lines of [kolibri.txt](https://github.com/learningequality/kolibri/files/4763888/kolibri.txt):
```
INFO 2020-06-11 17:28:46,551 django.server "GET /api/auth/session/current/?active=false HTTP/1.1" 200 201
INFO 2020-06-11 17:29:07,527 cherrypy.access 127.0.0.1 - - "GET /api/device/deviceprovision/" 401 0 "http://0.0.0.0:8000/en/setup_wizard/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:77.0) Gecko/20100101 Firefox/77.0"
WARNING 2020-06-11 17:29:07,527 django.server "GET /api/device/deviceprovision/?1591867747509=1591867747509 HTTP/1.1" 401 58
```

Last three (3) lines of [debug.txt](https://github.com/learningequality/kolibri/files/4763887/debug.txt):
```
INFO 2020-06-11 17:28:46,551 django.server "GET /api/auth/session/current/?active=false HTTP/1.1" 200 201
INFO 2020-06-11 17:29:07,527 cherrypy.access 127.0.0.1 - - "GET /api/device/deviceprovision/" 401 0 "http://0.0.0.0:8000/en/setup_wizard/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:77.0) Gecko/20100101 Firefox/77.0"
WARNING 2020-06-11 17:29:07,527 django.server "GET /api/device/deviceprovision/?1591867747509=1591867747509 HTTP/1.1" 401 58
```

Last three (3) lines of [2020-06-11-thu--yarn run devserver.txt](https://github.com/learningequality/kolibri/files/4763951/2020-06-11-thu--yarn.run.devserver.txt):
```
INFO     "GET /api/auth/session/current/?active=false HTTP/1.1" 200 201
INFO     127.0.0.1 - - "GET /api/device/deviceprovision/" 401 0 "http://0.0.0.0:8000/en/setup_wizard/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:77.0) Gecko/20100101 Firefox/77.0"
WARNING  "GET /api/device/deviceprovision/?1591867747509=1591867747509 HTTP/1.1" 401 58
```

Last three (6) lines of Firefox console output
[console-export-2020-6-11_18-9-52.txt](https://github.com/learningequality/kolibri/files/4764106/console-export-2020-6-11_18-9-52.txt) -- This one got different timestamps because I re-did the whole process, and got the same results.
```
---> <KCircularLoader> at node_modules/kolibri-design-system/lib/loaders/KCircularLoader.vue
       <SubmissionStatePage> at kolibri/plugins/setup_wizard/assets/src/views/submission-states/SubmissionStatePage.vue
         <ErrorPage> at kolibri/plugins/setup_wizard/assets/src/views/submission-states/ErrorPage.vue
           <Root> vue.runtime.esm.js:619
18:09:24.617 Error: Request failed with status code 401 createError.js:16
```

Here's the `~/.kolibri/plugins.json` content for reference (I cannot attach the file on Github):
```
{
  "DISABLED_PLUGINS": [],
  "INSTALLED_PLUGINS": [
    "kolibri.plugins.slideshow_viewer",
    "kolibri.plugins.html5_viewer",
    "kolibri.plugins.media_player",
    "kolibri.plugins.coach",
    "kolibri.plugins.epub_viewer",
    "kolibri_exercise_perseus_plugin",
    "kolibri.plugins.user",
    "kolibri.plugins.facility",
    "kolibri.plugins.app",
    "kolibri.plugins.learn",
    "kolibri.plugins.pdf_viewer",
    "kolibri.plugins.setup_wizard",
    "kolibri.plugins.h5p_viewer",
    "kolibri.plugins.device",
    "kolibri.plugins.default_theme"
  ],
  "PLUGIN_VERSIONS": {
    "kolibri.plugins.app": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.coach": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.default_theme": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.device": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.epub_viewer": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.facility": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.h5p_viewer": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.html5_viewer": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.learn": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.media_player": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.pdf_viewer": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.setup_wizard": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.slideshow_viewer": "0.14.0.dev0+git.20200610181821",
    "kolibri.plugins.user": "0.14.0.dev0+git.20200610181821",
    "kolibri_exercise_perseus_plugin": "1.3.2"
  },
  "UPDATED_PLUGINS": []
}
```

…

### Steps to reproduce

1. Delete the `~/.kolibri/` folder - or make a backup.
1. Follow the instructions to [Setting up Kolibri for development](https://kolibri-dev.readthedocs.io/en/develop/getting_started.html#setting-up-kolibri-for-development)
1. Run Kolibri, complete the **"Quick start"** Setup Wizard form then submit it.
1. Notice the Authentication Required prompt
![Screen Shot 2020-06-11 at 5 29 27 PM](https://user-images.githubusercontent.com/175580/84369755-e54e0480-ac09-11ea-8708-c6eceabd748e.png)

Here's the screenshot with the Firefox Console output when I cancel that authentication dialog:
![Screen Shot 2020-06-11 at 6 02 00 PM](https://user-images.githubusercontent.com/175580/84372508-c94c6200-ac0d-11ea-93de-0eebacbcef8d.png)

### Context
* Kolibri version: `develop` and `release-0.14.x`
* Operating system: macOS v10.14.6
* Browser: Firefox v77.0.1
* Python: v3.7.7

**Comment by mrpau-richard** [comment]
I have different error after I followed the steps to reproduce this issue.

The `IndexError` show after I run the `yarn run devserver`

here's the error logs
```
June 11, 2020 - 19:54:38
Django version 1.11.29, using settings 'kolibri.deployment.default.settings.dev'
Starting development server at http://0.0.0.0:8000/
Quit the server with CONTROL-C.
 OK
  Applying device.0006_migrate_guest_access...ERROR    Internal Server Error: /en/user/
Traceback (most recent call last):
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/Users/golf/.virtualenvs/kolibri-JVIbTfth/lib/python3.7/site-packages/django/views/decorators/http.py", line 88, in inner
    res_etag = etag_func(request, *args, **kwargs) if etag_func else None
  File "/Users/golf/kolibri/kolibri/core/decorators.py", line 323, in calculate_spa_etag
    + str(ThemeHook.cacheKey()).encode("utf-8")
  File "/Users/golf/kolibri/kolibri/core/theme_hook.py", line 140, in cacheKey
    theme = list(cls.registered_hooks)[0].theme
IndexError: list index out of range
 OK
  Applying device.0007_devicesettings_name... OK
  Applying device.0007_deviceappkey... OK
  Applying device.0008_merge_20200531_1829... OK
```

**Comment by cpauya** [comment]
Fixed by #7034 and tested on the [`release-v0.14.x`](https://github.com/learningequality/kolibri/tree/release-v0.14.x) branch.

The Setup Wizard now completes without prompts or errors.
![Screen Shot 2020-06-12 at 3 26 50 PM](https://user-images.githubusercontent.com/175580/84476836-4dabed00-acc1-11ea-9169-caab43f07476.png)

Thanks!
